src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannelFactory.cs

```diff
@@ -1,27 +1,28 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+// Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Reactive.Linq;
-using Microsoft.Azure.WebJobs.Script.Abstractions;
 using Microsoft.Azure.WebJobs.Script.Diagnostics;
 using Microsoft.Azure.WebJobs.Script.Eventing;
+using Microsoft.Azure.WebJobs.Script.Workers;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
```

Comment:
Does this namespace live in Script still?

Reply:
Yes, the abstractions `IRpcWorkerChannelFactory`, `IRpcWorkerChannel`, `IWorkerProcess` live in that namespace. The only files I have moved are the ones that have things specific to grpc. The RPC abstractions still live in `Webjobs.Script`.